### ISSUE TITLE
Exit code should not be 0 when failed.

### ISSUE THREAD

**Issue by sunshineo** [tutumcloud/dockup#4]
So I swapped my AWS key and forgot to update the configuration and this makes the tutum service silently exits.

It has error message:
2015-05-24T18:24:56.725961300Z tar: Removing leading `/' from member names
2015-05-24T18:25:00.812362200Z A client error (InvalidAccessKeyId) occurred when calling the ListBuckets operation: The AWS Access Key Id you provided does not exist in our records.
2015-05-24T18:25:01.622047200Z make_bucket failed: s3://gordonsun-blog/ A client error (InvalidAccessKeyId) occurred when calling the CreateBucket operation: The AWS Access Key Id you provided does not exist in our records.
2015-05-24T18:25:19.855023600Z upload failed: ./mariadb-backup.2015-05-24-18-24-56.tar.gz to s3://gordonsun-blog/mariadb-backup.2015-05-24-18-24-56.tar.gz A client error (InvalidAccessKeyId) occurred when calling the PutObject operation: The AWS Access Key Id you provided does not exist in our records.

But it is not possible to see on tutum log. According to Fernando the exit code was 0.
"It has an exit code of 0, so it didn't fail. Your service has autodestroy activated so it terminates containers when they finish. You can disable autodestroy and check the logs."

I think we need a different exit code when it fails to backup.

**Comment by girvo** [comment]
Has this been fixed? Worried about using it under a systemd timer if it doesn't have a proper exit code on failure

**Comment by sunshineo** [comment]
I don't think so. I'm not familiar with shell script. If you are, have a look at https://github.com/tutumcloud/dockup/blob/master/restore.sh and https://github.com/tutumcloud/dockup/blob/master/backup.sh
Can you make it return the exit code of the aws command call?
